src/index.html

```diff
@@ -7,6 +7,352 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link_active is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="">Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog">
+        <div class="card catalog__card" data-qa="card">
```

Comment:
```suggestion
        <div class="catalog__card card" data-qa="card">
```
usually it's written like this

Reply:
Got it.